docs/advanced/security.rst

```diff
@@ -0,0 +1,54 @@
+=================================
+Cluster Security & Authentication
+=================================
+
+A Flocker cluster comprises a control service and convergence agents, along with some command line tools that are provided to interact with and manage the cluster (see :doc:`architecture`).
+
+Flocker uses `Transport Layer Security <http://en.wikipedia.org/wiki/Transport_Layer_Security>`_ (TLS) to authenticate components of a cluster against each other in a `mutual authentication<http://en.wikipedia.org/wiki/Mutual_authentication>`_ model.
```

Comment:
The second link on this line appears to have a markup error.  Maybe due to lack of space between text and link?